.github/workflows/continuous-integration.yml

```diff
@@ -135,8 +131,8 @@ jobs:
       - name: "Create build directory"
         run: "mkdir -p .build/logs"
 
-      - name: "Collect code coverage with Xdebug and phpunit/phpunit"
-        run: "vendor/bin/phpunit --coverage-clover=.build/logs/clover.xml"
+      - name: "Collect code coverage with Xdebug and phpunit"
```

Comment:
Can we revert this unrelated change?

We are still running `phpunit/phpunit`.

Reply:
In my opinion this is clear based on the context given, but I will revert it.

Reply:
reverted